docs/docsource/Operation-Architecture/Zenko_CLI/Object_Lifecycle_Management.rst

```diff
@@ -98,4 +98,47 @@ for more examples and explanations on lifecycle rules.
        $ aws s3api get-bucket-lifecycle-configuration --bucket zenko-bucket
 
 Once the lifecycle rules on the bucket are set, the rules apply to all
-objects in the specified bucket.
\ No newline at end of file
+objects in the specified bucket.
+
+.. _Lifecycle Queries:
+
+Querying Lifecycle Events
+-------------------------
+
+A lifecycle metrics feature is not available for release 1.1. You can,
```

Comment:
> A lifecycle metrics feature is not available for release 1.1. 

I think we can leave this sentence out because there is a chance this might change. If we are able to implement metrics, I think the doc here will apply nicely still. 😸

Reply:
done